NewsBook/Application/Commands/FavouriteNews/FavouriteNewsCommands.cs

```diff
@@ -0,0 +1,18 @@
+﻿using MediatR;
+using NewsBook.Mediator.Response;
+
+namespace NewsBook.Mediator.Commands.FavouriteNewsCommands
+{
+    public class MarkFavouriteNews : IRequest<NewsReadDTO>
+    {
+        public Guid id;
+    }
+    public class PutFavouriteNews : IRequest<NewsReadDTO>
```

Comment:
The basic purpose of introducing the MediatR pattern reduce the dependency on the HTTP web API protocol.

Suffixing the commands using HTTP method names introduces the naming coupling with the HTTP protocol methods. Avoid it..